pages/neighbourhoods.py

```diff
@@ -0,0 +1,434 @@
+
+    return min, max, marks, value, list(range(min, max)), value
+
+
+@callback(
+    Output('map', 'figure'),
+    Output('title_map', 'children'),
+    Input('slider_select_year', 'value'),
+    Input('drop_var_id', 'value'),
+    Input('drop_municipality', 'value'),
+    Input('drop_municipality_spec_id', 'value')
+    )
+
+def update_graph_map(year_value, xaxis_column_name, wijk_name, wijk_spec
+                 ):
+    '''
+    Select the appropriate data to display in the map fig
+    '''
+    dff = df[df['YEAR'] == year_value]
+
+    title = '{} - {} - {} '.format(xaxis_column_name, wijk_name, year_value)
+        
+    dff = dff.query("Wijknaam in @wijk_spec")
+
+    fig = px.choropleth_mapbox(dff, geojson=geo_df, color=xaxis_column_name,
+                            locations="WKC", featureidkey="properties.WKC", opacity = 0.5,
+                            center={"lat": 52.0705, "lon": 4.3003}, color_continuous_scale=colorscale,
+                            mapbox_style="carto-positron", zoom=10, hover_name="Wijknaam")
+    
+    fig.update_layout(geo=dict(bgcolor= 'rgba(0,0,0,0)', lakecolor='#4E5D6C'),
+                                autosize=False,
+                                  font = {"size": 9, "color":"black"},
+                                  margin={"r":0,"t":10,"l":10,"b":50},
+                                  paper_bgcolor='white'
+                                  )
+    
+    return fig, title
+
+# create a new column that put each row into a group of 4 numbers based on the value of a column quartile
+
+@callback(
+    Output('title_bar', 'children'),
+    Output('bar_fig', 'figure'),
+    Input('slider_select_year', 'value'),
+    Input('drop_var_id', 'value'),
+    Input('drop_municipality', 'value'),
+    Input('drop_municipality_spec_id', 'value')
+    )
+
+def update_graph_bar(year_value, xaxis_column_name, wijk_name, wijk_spec):
+    '''
+    Update the bar chart based on new values
+    '''
+    dff = df[df['YEAR'] == year_value]
+    
+    if len(wijk_spec) == 0:
+        fig = px.bar(x=[0, 10],
+                y=[0, 0]
+                )
+        fig.update_xaxes(showticklabels=False)
+        fig.update_yaxes(showticklabels=False)
+        
+        return "No neighbourhood selected", fig
+
+    elif len(wijk_spec) < 3:
+        dff = dff.query("Wijknaam in @wijk_spec")
+        dff = dff.sort_values(by=[xaxis_column_name], ascending=False).reset_index()    
+        fig = px.bar(dff, xaxis_column_name, 'Wijknaam',
+                hover_name='Wijknaam', color_discrete_sequence=get_colors(0, 1))
+        
+    else:
+        dff = dff.query("Wijknaam in @wijk_spec")
+        dff = dff.sort_values(by=[xaxis_column_name], ascending=False).reset_index()   
+        dff['group'] = pd.qcut(dff[xaxis_column_name], 3, labels=['Low', 'Medium', 'High'])
+
+        fig = px.bar(dff, xaxis_column_name, 'Wijknaam', color= 'group',
+                hover_name='Wijknaam', color_discrete_sequence=get_colors(0.2, 3))
```

Comment:
I have an error on this categorization. I think we can just remove these. I used it to get strict cut off points for the color before. But I think we don't need it anymore. What do think?